src/components/ShardChainsList.tsx

```diff
@@ -102,9 +98,9 @@ const ShardChainsList: React.FC<IProps> = () => {
   ]
 
   return (
-    <Container>
+    <Box as="div" mb="4rem">
```

Comment:
Same as the `MergeArticleList` migration PR, `Box` renders `div` by default.
```suggestion
    <Box mb="4rem">
```